resources/settings.xml

```diff
@@ -4,6 +4,7 @@
     <setting id="lbry_api_url" type="text" option="urlencoded" label="30000" default="https://api.lbry.tv/api/v1/proxy"/>
     <setting id="nsfw" type="bool" label="30001" default="false"/>
     <setting id="items_per_page" type="number" label="30002" default="20"/>
+    <setting id="user_channel" type="text" label="User channel" default=""/>
```

Comment:
I think its better to remove this as a edit field, and just keep this value internal, getting the possible user-channels from the API server